src/main/resources/templates/snowflake/snowflake-schema-evolution/Makefile.ssp

```diff
@@ -0,0 +1,15 @@
+#import(io.phdata.streamliner.schemadefiner.model.Snowflake)
```

Comment:
Please review this Makefile for evolve schema.

Reply:
All, I see we are creating an entirely new set of templates. I guess I can see a statement template set. For the stand alone template set:

1. first-run should just do create-table.
2. Then run should call evolve-schema.

For example, snowflake-aws-dms-merge should be updated as the first step in the run method:

https://github.com/phdata/streamliner/blob/master/src/main/resources/templates/snowflake/snowflake-aws-dms-merge/Makefile.ssp#L95

then for the append templates, we should create a target run which calls evolve-schema and then copy-into.

https://github.com/phdata/streamliner/blob/master/src/main/resources/templates/snowflake/snowflake-snowpipe-append/Makefile.ssp#L51

Reply:
Sorry but I could not get all these points. One question raised. For schema-evolution we have created a separate folder snowflake-schema-evolution that will have evolve-schema.sql.ssp, Makefile.ssp and Makefile-configDiff.schema.ssp(https://github.com/phdata/streamliner/pull/43/commits/2b8980aea51739d28f9113478dbdbd0c1a6c0786#diff-ac912d4eb262c06498244064a0eb2bb536d4c9b22d5c4f4bc9d940692a279690). No other folder will be accessed during schema evolution. Then how append template will get evolve-schema

Reply:
I was making this too complex.  Let's ignore the other snowflake templates for now.
 The use case for "schema-evolution" is to keep the tables in sync between a source and destination.

Too make that happen three cases need to be handled:

1. First time the code runs, all tables in source need to be defined.
2. Next run and every run after both new columns need to be added to existing tables and new tables need to be defined.

Reply:
updated https://github.com/phdata/streamliner/pull/43/commits/f38fa958f867d43a0936f6eeb61b1211435e8523#diff-84da8a6e57c39c21dbc4254df61c7963ed7fbf3b3859b27e4bc9073b05a60c84